src/netmanager-app/core/hooks/useMapData.ts

```
@@ -0,0 +1,32 @@
+// hooks/useMapboxData.ts
+"use client"; 
+import { useEffect, useState } from "react";
+import {GetAirQuoData} from "../apis/MapData"
+
+export function useMapData() {
+  const [MapData, setMapData] = useState(null);
+  const [Error, setError] = useState<string | null>(null);
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)